frontend/cypress/pages/metrics/metrics.ts

```diff
@@ -42,6 +44,13 @@ class Metrics {
     cy.get('[data-test-id="remove-button"]').eq(1).click()
   }
 
+  checkJsonFile() {
+    cy.contains('Save').click()
+    cy.readFile('./cypress/downloads/config.json').then((fileContent) => {
+      expect(fileContent).to.deep.equal(JSON_FILE_CONTENT)
```

Comment:
Learn [this part](https://app.mural.co/t/twma7655/m/twma7655/1669606203264/a021d08cf2e5f242004dac8baf7c64d40d7552c4?wid=0-1681104777602),

then re-write this method

Reply:
I will learn POM again and pick a new pr for this e2e test